docs/types/array.md

```diff
@@ -124,28 +154,15 @@ Returns a new array with only the elements that returned
 ["hello", 0, 1, 2].filter(f(x){type(x) == "NUMBER"}) # [0, 1, 2]
 ```
 
-### json()
-
-Parses the string as JSON, returning an [hash](/types/hash):
-
-``` bash
-"{}".json() # {}
-```
-
-Note that currently only JSON objects are supported,
```

Comment:
this should be removed though, as these issues have been fixed

Reply:
Ok, i'll fix this.